tests/test_parsers.py

```diff
@@ -93,6 +93,39 @@ class Flake8ParserTestCase(ParserTestCaseMixin, unittest.TestCase):
     }
 
 
+class BanditJSONParserTestCase(ParserTestCaseMixin, unittest.TestCase):
+    parser = PARSERS['bandit-json']
+    linter_output_file_name = 'bandit-json.txt'
+    expected_violations = {
+        'build/lib/lintly/formatters.py': [
+            {'line': 14, 'column': 0, 'code': 'B701 (jinja2_autoescape_false)',
+             'message': 'Using jinja2 templates with autoescape=False is dangerous and can lead to XSS.
+             Use autoescape=True or use the select_autoescape function to mitigate XSS vulnerabilities.'}
```

Comment:
Formatting nit: for one, this is hard to read -- it's hard to tell from a glance that this line is inside a quote. And a more practical issue -- this will put a newline and a bunch of spaces after the '...XSS.'

One of the standard ways to deal with very long strings is to use python's implicit string concatenation. ('`'a' ' ' 'word'` -> `'a word'`)
```suggestion
             'message': ('Using jinja2 templates with autoescape=False is dangerous and can lead to XSS.'
                         'Use autoescape=True or use the select_autoescape function to mitigate XSS vulnerabilities.')}
```